NugetLightRequest.cs

```diff
@@ -1587,7 +1612,13 @@ into newGroup
                     }
                     else
                     {
-                        pkgs = from p in pkgs where p.IsLatestVersion select p;
+                        SemanticVersion latest = new SemanticVersion("0.0.0");
```

Comment:
I don't quite understand why this bit has changed. Is the .IsLatestVersion property not giving the right answer here?

Reply:
Yeah unfortunately it doesn’t.  .isLatestVersion is only true for the latest version of a module.  This function filters our packages based on parameter input and so there may not be a true .IsLatestVersion property in the filtered subset.